ray-operator/test/support/yaml.go

```diff
@@ -55,3 +55,19 @@ func KubectlApplyYAML(t Test, filename string, namespace string) {
 	assert.NoError(t.T(), err)
 	t.T().Logf("Successfully applied %s", filename)
 }
+
+func KubectlApplyQuota(t Test, namespace, quota string) {
+	t.T().Helper()
+	kubectlCmd := exec.CommandContext(t.Ctx(), "kubectl", "create", "quota", namespace, "-n", namespace, quota)
+	err := kubectlCmd.Run()
+	assert.NoError(t.T(), err)
```

Comment:
`assert.NoError` does not fail immediately; the error is only printed after the test finishes and `"Successfully applied quota %s in %s"` will always be printed no matter the command succeed or not. Should we make it fail fast?